src/module.ts

```diff
@@ -184,158 +184,161 @@ export default defineNuxtModule<ModuleOptions>().with({
-      config.externals.inline ||= []
-      config.externals.inline.push(...[
-        resolve('runtime/utils'),
-        resolve('runtime/form-data'),
-        resolve('runtime/server/$api'),
-      ])
-
-      // Provide `#nuxt-api-party/server` module alias for Nitro
-      config.alias ||= {}
-      config.alias[`#${moduleName}/server`] = resolve(nuxt.options.buildDir, `module/${moduleName}-nitro`)
-
-      config.virtual ||= {}
-      config.virtual[`#${moduleName}/server`] = () => `
+        config.externals ||= {}
+        config.externals.inline ||= []
+        config.externals.inline.push(...[
+          resolve('runtime/utils'),
+          resolve('runtime/form-data'),
+          resolve('runtime/server/$api'),
+        ])
+
+        // Provide `#nuxt-api-party/server` module alias for Nitro
+        config.alias ||= {}
+        config.alias[`#${moduleName}/server`] = resolve(nuxt.options.buildDir, `module/${moduleName}-nitro`)
+
+        config.virtual ||= {}
+        config.virtual[`#${moduleName}/server`] = () => `
 import { _$api } from '${resolve('runtime/server/$api')}'
 ${endpointKeys.map(i => `
 export const ${getRawComposableName(i)} = (...args) => _$api('${i}', ...args)
 `.trimStart()).join('')}`.trimStart()
 
-      if (endpointKeys.length) {
-        config.typescript ||= {}
-        config.typescript.tsConfig ||= {}
-        config.typescript.tsConfig.include ||= []
-        config.typescript.tsConfig.include.push(`./module/${moduleName}-schema.d.ts`)
-        config.typescript.tsConfig.include.push(`./module/${moduleName}-hooks.d.ts`)
-      }
-
-      // Add Nitro auto-imports for generated composables
-      config.imports = defu(config.imports, {
-        presets: [{
-          from: `#${moduleName}/server`,
-          imports: endpointKeys.map(getRawComposableName),
-        }],
+        if (endpointKeys.length) {
+          config.typescript ||= {}
+          config.typescript.tsConfig ||= {}
+          config.typescript.tsConfig.include ||= []
+          config.typescript.tsConfig.include.push(`./module/${moduleName}-schema.d.ts`)
+          config.typescript.tsConfig.include.push(`./module/${moduleName}-hooks.d.ts`)
+        }
+
+        // Add Nitro auto-imports for generated composables
+        config.imports = defu(config.imports, {
+          presets: [{
+            from: `#${moduleName}/server`,
+            imports: endpointKeys.map(getRawComposableName),
+          }],
+        })
       })
-    })
 
-    // Add Nuxt auto-imports for generated composables
-    addImportsSources({
-      from: resolve(nuxt.options.buildDir, `module/${moduleName}`),
-      imports: endpointKeys.flatMap(i => [getRawComposableName(i), getDataComposableName(i)]),
-    })
+      // Add Nuxt auto-imports for generated composables
+      sources.push({
+        from: resolve(nuxt.options.buildDir, `module/${moduleName}`),
+        imports: endpointKeys.flatMap(i => [getRawComposableName(i), getDataComposableName(i)]),
+      })
```

Comment:
Can't use the helper here because it's called while already in the `import:sources` hook.